### ISSUE TITLE
Reduce odds of Panel Buckling via stringers or wing box

### ISSUE THREAD

**Issue by sean-engelstad** [smdogroup/caps2fun#6]
### Issue Type

- [ ] Bug
- [ ] Enhancement

### Description

Keep getting panel buckling solutions, with OML panels near the root sometimes buckling. We are still experiencing panel buckling solutions even for reasonably large thicknesses near the root of the wing. Will try adding stringers and maybe also add a wing box so that the hard cantilever constraint is not applied at the root rib. 

### Example

NacaWing

**Comment by sean-engelstad** [comment]
I was tinkering with adding boosts to the membrane and bending stiffnesses of the OML (boosting as if the stringers were there since I couldn't get the mesher to mesh them yet). I got a pretty good response in the structural analysis now (10x scale factor deformation below), and about 0.45 ksfailure vs 15 before.
![structResponse](https://user-images.githubusercontent.com/89812682/168909275-f198c955-22f3-4d96-b5b2-ac3a0c58b610.png)
![wingDeform](https://user-images.githubusercontent.com/89812682/168909284-37b6dabd-04ac-4070-a5ae-16195b879f97.png)

Q: What do you mean by "boosts to the membrane and bending stiffnesses"?
Well, if you have stringers they are perpendicular to the OML/vertical so they provide much more bending stiffness which are H=EI and I=1/12*w*t^3 for the OML panel but for the stringers it's 1/12*t*h^3 where h is a good bit larger than t. Therefore you get much more bending stiffness H. In CAPS you provide a shell property card which has the material type, membrane thickness, shear membrane ratio, and bending inertia ratio. I took the default settings and multiplied them by 20.0 for shear membrane and bending inertia ratios for only the OML shell property cards. I decided to do it that way since we don't have the stringers in the geometry right now, we still need the membrane and shear stiffnesses to be more realistic as if there were stringers tehre

**Comment by sean-engelstad** [comment]
For reference, before the artificial boosts to shear and membrane stiffnesses of the OML panels, the OML panels near the root of the wing would buckle. They just had way too low bending and shear membrane stiffnesses despite having pretty large thicknesses. The issue was they had n
![Screenshot from 2022-05-15 12-02-13](https://user-images.githubusercontent.com/89812682/168909685-51c68a9f-8f23-43ef-84ae-390ee6d9f868.png)
o stringers or artificial boost to their bending stiffnesses, so they were too likely to buckle.

**Comment by sean-engelstad** [comment]
I had tried to add stringers and managed to do so at least in the .csm file geometry by making a smaller wing with lower tc0, tcf in the naca udprim. Then I subtract the smaller wing from the temporary stringer spars and union with the resulting wing to make the stringers. However, the edges on the root and tip rib have sharp edges and therefore the automated structure mesher fails. For this reason the stringers can be toggled on and off and are default off for now until further improvement.
![stringers](https://user-images.githubusercontent.com/89812682/168910128-ee73dcd0-fca0-4de2-9f00-d6fd384daf01.png)

**Comment by sean-engelstad** [comment]
I have added a boost factor setting in the funtofem.cfg files that increases bending stiffnesses of the OML capsGroup/panels only. This artifically boosts the bending stiffnesses as if stringers were there. I was able to get more reasonable ksfailure, lift, and drag values sometimes after that. Although, I have a new issue now with ksfailure and temperature. However, I don't think I actually need stringers or a wing box now.
